pilot/pkg/xds/monitoring.go

```diff
@@ -118,6 +118,12 @@ var (
 	ldsSendErrPushes = pushes.With(typeTag.Value("lds_senderr"))
 	rdsSendErrPushes = pushes.With(typeTag.Value("rds_senderr"))
 
+	pushContextInitTime = monitoring.NewDistribution(
+		"pilot_pushcontext_init_time",
+		"Total time in seconds Pilot takes to init pushContext.",
+		[]float64{.01, .1, 1, 3, 5, 10, 20, 30},
```

Comment:
`3, 5, 10, 20, 30` buckets seem pretty large, maybe we should have some more smaller buckets and less large ones?

Reply:
SG